pages/500.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import Error from '../components/Error'
+
+export default function NotFound() {
```

Comment:
A status code of `500` is a server error and isn't `not found`. This function name is misleading.

Reply:
Looks reasonable, will be fixed.

Reply:
Fixed.